dpc-go/dpc-attribution/Makefile

```
@@ -0,0 +1,15 @@
+.PHONY: unit-test
+unit-test:
+	@bash ./unit_test.sh
+
+.PHONY: lint
+lint:
+	@echo "-------------- ATTRIBUTION LINTING STARTED --------------"
+	@golangci-lint run && echo "*********** ATTRIBUTION IS LINT FREE!! ***********" || echo "*********** LINTING FAILED!! ***********"
+	@echo "-------------- SECURITY SCAN STARTED --------------"
+	@cd src && gosec -fmt=json -out=../results.json ./... && cd ..
```

```suggestion
	@cd src && gosec -fmt=json -out=../results.json ./...
```
At least in my shell, a `cd` command does not affect the following line